Formula/f3d.rb

```diff
@@ -0,0 +1,48 @@
+class F3d < Formula
+  desc "Fast and minimalist 3D viewer"
+  homepage "https://kitware.github.io/F3D/"
+  url "https://gitlab.kitware.com/f3d/f3d/-/archive/v1.0.1/f3d-v1.0.1.tar.gz"
+  sha256 "fb362dba3ccf49db9e8841d8a5310f37399bfea8866b6e040ce85670d54b97f7"
+  license "BSD-3-Clause"
+
+  depends_on "cmake" => :build
+  depends_on "vtk"
+
+  def install
+    # Is fixed upstream, remove this line for v1.1
+    inreplace "src/F3DOptions.cxx",
+    "directoryPath = vtksys::SystemTools::GetProgramPath(programFilePath);",
+    "programFilePath = vtksys::SystemTools::GetRealPath(programFilePath);"\
+    "directoryPath = vtksys::SystemTools::GetProgramPath(programFilePath);"
```

Comment:
This might just be my personal preference (so feel free to ignore) but it would be good if you could indent this, and add a space before the `\` in the second line.

A `+` may also be more appropriate (instead of the `\`), but I don't really speak Ruby, so I could also be wrong on this one. It just strikes me as conveying string concatenation more clearly.

Reply:
Whoops. I was wrong about `+`.

Sorry!

Reply:
no problem ;)  
Fixed!